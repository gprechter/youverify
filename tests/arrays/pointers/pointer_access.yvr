p: INT*
result: INT
p = ARRAY[3]{0}
*p = 1
result = *p