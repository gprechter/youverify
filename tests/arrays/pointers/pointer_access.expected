{
"p": [1,0,0],
"result": 1
}